teaser/logic/buildingobjects/thermalzone.py

```diff
@@ -477,18 +479,30 @@ def name(self):
 
     @name.setter
     def name(self, value):
+        regex = re.compile('[^a-zA-z0-9]')
+        # name = None
         if isinstance(value, str):
-            regex = re.compile('[^a-zA-z0-9]')
-            self._name = regex.sub('', value)
+            # self._name = regex.sub('', value)
```

Comment:
Can we delete the comments?

Reply:
fixed with 9859fd0448812017ff63fbafdcd68d7dfea9fcd2